src/cdk-experimental/menu/focus-mouse-manager.ts

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
I think I would change the name of this file to something like `item-pointer-entries.ts`

Reply:
Done